packages/xo-server-sdn-controller/README.md

```diff
@@ -0,0 +1,43 @@
+# xo-server-sdn-controller [![Build Status](https://travis-ci.org/vatesfr/xen-orchestra.png?branch=master)](https://travis-ci.org/vatesfr/xen-orchestra)
+
+XO Server plugin that allows the creation of pool-wide private networks.
+
+## Install
+
+For installing XO and the plugins from the sources, please take a look at [the documentation](https://xen-orchestra.com/docs/from_the_sources.html).
+
+## Usage
+
+### Network creation
+
+In the network creation view, select a `pool` and `Private network`.
+Create the network.
+
+Choice is offer between `GRE` and `VxLAN`, if `VxLAN` is chosen, then the port 4789 must be open for UDP traffic.
+The following line needs to be added, if not already present, in `/etc/sysconfig/iptables` of all the hosts where `VxLAN` is wanted:
+`-A xapi-INPUT -p udp -m conntrack --ctstate NEW -m udp --dport 4789 -j ACCEPT`
+
+### Configuration
+
+Like all other xo-server plugins, it can be configured directly via
+the web interface, see [the plugin documentation](https://xen-orchestra.com/docs/plugins.html).
+
+The plugin's configuration contains:
+- `cert-dir`: A path where to find the certificates to create SSL connections with the hosts.
+If none is provided, the plugin will create its own self-signed certificates.
+- `override-certs:` Whether or not to uninstall an already existing SDN controller CA certificate in order to replace it by the plugin's one.
```

Comment:
This file is not visible by users and is not the correct place to add documentation.

If you want to write some doc, do so in https://github.com/vatesfr/xen-orchestra/tree/master/docs

Reply:
I'm not sure its necessary in fact.
Since the description is already present in the UI of the plugin settings.

Reply:
Fair enough

Reply:
Should I remove it from the README.md?

Reply:
Not necessary, that will server as dev documentation :slightly_smiling_face: